### PR TITLE
Feat/annotationsearch

### PR DESCRIPTION
**Pull request by OzgeYurtsever**

### CODE REVIEW

**Review by muakdogan — COMMENTED** [review]
annotations.jsx --> case sensitivity

File: src/components/annotationSearch/index.jsx [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+
+const lists = {
+  organize: ['AND', 'OR', '(', ')'],
+  paranthesis: ['(', ')'],
+  condition: ['AND', 'OR'],
+  type: ['modality', 'diagnosis', 'anatomy'],
+  criteria: ['equals', 'contains']
+};
+
+const title = {
```
Comment (muakdogan) [inline src/components/annotationSearch/index.jsx:22]: Do you need this?

File: src/components/annotationSearch/index.jsx [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+    );
+  };
+
+  const addPartialToQuery = () => {
+    const { type, criteria, term } = partialQuery;
+    const newQuery = `${query} ${type} ${criteria} ${term}`;
+    setQuery(newQuery);
+  };
+
+  const insertIntoQuery = selection => {
```
Comment (muakdogan) [inline src/components/annotationSearch/index.jsx:123]: delete?

File: src/components/annotationSearch/index.jsx [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+
+  // finds the index of firt predefined term
+  const detactTermEndIndex = baseQuery => {
+    const { organize, paranthesis, type, criteria } = lists;
+    const indexMap = {};
+    const list = _.concat(organize, paranthesis, type, criteria);
+    list.forEach(el => {
+      indexMap[el] = baseQuery.indexOf(el);
+    });
+    // cover if query ends there ??????
```
Comment (muakdogan) [inline src/components/annotationSearch/index.jsx:214]: todo???

**Review by muakdogan — COMMENTED** [review]
second test review

File: src/components/annotationSearch/index.jsx (hunk 1 above)
Comment (OzgeYurtsever) [reply src/components/annotationSearch/index.jsx:22]: deleted!

File: src/components/annotationSearch/index.jsx (hunk 2 above)
Comment (OzgeYurtsever) [reply src/components/annotationSearch/index.jsx:123]: deleted!

File: src/components/annotationSearch/index.jsx (hunk 3 above)
Comment (OzgeYurtsever) [reply src/components/annotationSearch/index.jsx:214]: This method is not used. Deleted!

**Pull request merged**